schemas/2015-01-01/deploymentTemplate.json

```diff
@@ -579,7 +579,8 @@
                   { "$ref": "https://schema.management.azure.com/schemas/2016-12-01/Microsoft.RecoveryServices.json#/resourceDefinitions/vaults_backupPolicies" },
                   { "$ref": "https://schema.management.azure.com/schemas/2017-07-01/Microsoft.RecoveryServices.json#/resourceDefinitions/vaults_backupFabrics_backupProtectionIntent" },
                   { "$ref": "https://schema.management.azure.com/schemas/2018-04-01/Microsoft.Compute.json#/resourceDefinitions/disks" },
-                  { "$ref": "https://schema.management.azure.com/schemas/2018-04-01/Microsoft.Compute.json#/resourceDefinitions/snapshots" }
+                  { "$ref": "https://schema.management.azure.com/schemas/2018-04-01/Microsoft.Compute.json#/resourceDefinitions/snapshots" },
+                  { "$ref": "https://schema.management.azure.com/schemas/2018-04-01/Microsoft.ContainerInstance.json#/resourceDefinitions/containerGroups" }
```

Comment:
>containerGroups [](start = 138, length = 15)

add to 2014-04-01-preview/deploymentTemplate.json as well?

Reply:
thanks! Added.